cpu/mips_pic32mx/Makefile.include

```diff
@@ -1,3 +1,5 @@
+CPU_ARCH = m4k
```

Comment:
I would prefer CPU_ARCH/CPU_FAM set in `Makefile.features`, same for `mips_pic32mz`

Reply:
I put it in these files because when I grep CPU_ARCH/CPU_FAM, they are always defined in `Makefile.include`.

Reply:
After #13236 and #13174 are merged this will no longer be the case ;)

Reply:
done